pkg/interop/json/json.go

```diff
@@ -0,0 +1,14 @@
+/*
+Package json provides various JSON serialization/deserialization routines.
+*/
+package json
+
+// ToJSON serializes value to json. It uses `System.Json.Serialize` syscall.
```

Comment:
I think the serialization format should be described here, how to use this if I don't know what's going to happen to my stack items?

Reply:
Fixed.